src/server.ts

```diff
@@ -3,13 +3,11 @@ import * as express from 'express'
 
 import errorMiddleware from 'error-middleware'
 
-import * as dotenv from 'dotenv'
 import * as masterRouter from './api'
-
-dotenv.config()
+import { server } from './config/config'
 
 const app: express.Application = express()
-const port: string = process.env.PORT || '3000'
+const port: string = server.port || '3000'
```

Comment:
aren't we doing this `OR` clause in the config already? it seems to be duplicated. correct me if i'm wrong